polymorphic_auth/admin.py

```diff
@@ -8,6 +9,56 @@
 from polymorphic_auth.models import User
 from polymorphic.admin import \
     PolymorphicParentModelAdmin, PolymorphicChildModelAdmin
+from . import plugins
+
+
+class ChildModelPluginPolymorphicParentModelAdmin(PolymorphicParentModelAdmin):
+    """
+    Get child models and choice labels from registered plugins.
+    """
+
+    child_model_plugin_class = None
+    child_model_admin = None
+
+    def __init__(self, *args, **kwargs):
+        self.child_model_plugin_class = plugins.PolymorphicAuthChildModelPlugin
+        super(ChildModelPluginPolymorphicParentModelAdmin, self).__init__(
+            *args, **kwargs)
+
```

Comment:
Is this necessary? The `child_model_plugin_class` attribute should just be declared on subclasses, e.g. on `UserAdmin`?